plugins/woocommerce-blocks/bin/webpack-configs.js

```diff
@@ -378,20 +383,17 @@ const getFrontConfig = ( options = {} ) => {
 				minSize: 200000,
 				automaticNameDelimiter: '--',
 				cacheGroups: {
-					...getCacheGroups(),
-					'base-components': {
```

Comment:
This was duplicated from an identical entry generated by `getCacheGroups` so this is tidy up.